src/main/java/uk/gov/digital/ho/hocs/casework/api/utils/ActionDataHelpers.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.digital.ho.hocs.casework.api.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActiveStage;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+@Slf4j
+public class ActionDataHelpers {
+    private ActionDataHelpers() {}
+
+    // COPIED FROM CaseDataService to avoid cyclic dependency.
+    public static void updateStageDeadlines(CaseData caseData) {
```

Comment:
This is probably better to implement as non-void and include tests written against it

Reply:
I've added a test, however I've left the method as void, it isn't necessary to return an object for the test and I think leaving as void is a clearer indication that we are manipulating an existing object rather than returning the passed in object, that from the outside looks like creating a new CaseData object.